### PR TITLE
Fix all staticcheck errors

### PR DESCRIPTION
**Pull request by taveek**

### CODE REVIEW

**Comment by taveek** [comment]
Please review and merge, thanks

File: cmd/commands/dlv/dlv_amd64.go [hunk 1]
```diff
@@ -227,7 +227,7 @@ func startWatcher(paths []string, ch chan int) {
 
 					// Wait 1s before re-build until there is no file change
 					scheduleTime := time.Now().Add(1 * time.Second)
-					time.Sleep(scheduleTime.Sub(time.Now()))
+					time.Until(scheduleTime)
```
Comment (amrfaissal) [inline cmd/commands/dlv/dlv_amd64.go]: What happened to `time.Sleep`?

File: cmd/commands/run/watch.go [hunk 2]
```diff
@@ -85,7 +85,7 @@ func NewWatcher(paths []string, files []string, isgenerate bool) {
 					go func() {
 						// Wait 1s before autobuild until there is no file change.
 						scheduleTime = time.Now().Add(1 * time.Second)
-						time.Sleep(scheduleTime.Sub(time.Now()))
+						time.Until(scheduleTime)
```
Comment (amrfaissal) [inline cmd/commands/run/watch.go]: What happened to `time.Sleep`?

File: cmd/commands/dlv/dlv_amd64.go (hunk 1 above)
Comment (taveek) [reply cmd/commands/dlv/dlv_amd64.go]: oops! you are right. I'm gonna fix it.

File: cmd/commands/run/watch.go (hunk 2 above)
Comment (taveek) [reply cmd/commands/run/watch.go]: Sorry about mistake.

**Pull request merged**